doc/application_workflow/harbor_registry/harbor_with_static_ip.md

```diff
@@ -0,0 +1,209 @@
+# Installing Harbor with static IP for use with vSphere Integrated Container Engine
+
+For testing and development purposes, it is often useful to simply deploy Harbor with a static IP.  This workflow will demonstrate how to do that.  The primary difference in this workflow versus the one for FQDN will be in the vApp parameters for Harbor and the certificate generation section.  When installing Harbor with static IP, we will install Harbor with networking and the certificate in one step instead of the two steps that we listed for FQDN.  Note, our install for FQDN required two steps to accomodate our setup.  other VC setups may only require 1 step.
+<br><br>
+
+## Workflow
+
+We will use VIC Engine 0.8.0 and Harbor 0.4.5 for this example.  We will also use Windows during the install of Harbor as we will use the vSphere thick client for Windows.  We will use Ubuntu as OS on our user machine for every other step.
+
+By the end of this, we should have demonstrated,
```

Comment:
Minor:  Something like  `By the end of this example, we will have demonstrated the following:`